Dockerfile

```diff
@@ -47,3 +60,9 @@ RUN echo $DEV_USER:$DEV_USER_PASSWD | chpasswd
 
 # Initialize development environment for $DEV_USER.
 RUN sudo -u $DEV_USER -H git config --global credential.helper 'cache --timeout=3600'
+
+# Install ostreeuploader, aka fiopush/fiocheck
+COPY --from=0 /ostreeuploader/bin/fiopush /usr/bin/
```

Comment:
Better to update to use a named stage instead of 0, to avoid possible issue in the future related to the order of the changes.

Reply:
updated